pkg/configure/oneagent/conf/content.go

```diff
@@ -18,39 +54,89 @@ type fileContent struct {
 	ImageName               string `json:"imageName"`
 }
 
-func (c fileContent) toMap() (map[string]string, error) {
-	return structs.ToMap(c)
+func (cs containerSection) toMap() (map[string]string, error) {
+	return structs.ToMap(cs)
 }
 
-func (c fileContent) toString() (string, error) {
-	var confContent strings.Builder
+func (cs containerSection) toString() (string, error) {
+	var sectionContent strings.Builder
 
-	contentMap, err := c.toMap()
+	contentMap, err := cs.toMap()
 	if err != nil {
 		return "", err
 	}
 
-	confContent.WriteString("[container]")
-	confContent.WriteString("\n")
+	sectionContent.WriteString("[container]")
+	sectionContent.WriteString("\n")
 
 	for key, value := range contentMap {
-		confContent.WriteString(key)
-		confContent.WriteString(" ")
-		confContent.WriteString(value)
-		confContent.WriteString("\n")
+		if value == "" {
+			continue
+		}
+
+		sectionContent.WriteString(key)
+		sectionContent.WriteString(" ")
+		sectionContent.WriteString(value)
+		sectionContent.WriteString("\n")
 	}
 
-	return confContent.String(), nil
+	return sectionContent.String(), nil
+}
+
+type hostSection struct {
+	Tenant      string `json:"tenant"`
+	IsFullStack string `json:"isCloudNativeFullStack"`
+}
+
+func (hs hostSection) toMap() (map[string]string, error) {
+	return structs.ToMap(hs)
+}
+
+func (hs hostSection) toString() (string, error) {
+	var sectionContent strings.Builder
+
+	contentMap, err := hs.toMap()
+	if err != nil {
+		return "", err
+	}
+
+	sectionContent.WriteString("[host]")
+	sectionContent.WriteString("\n")
+
+	for key, value := range contentMap {
+		if value == "" {
+			continue
+		}
+
+		sectionContent.WriteString(key)
+		sectionContent.WriteString(" ")
+		sectionContent.WriteString(value)
+		sectionContent.WriteString("\n")
+	}
+
+	return sectionContent.String(), nil
 }
 
-func fromAttributes(containerAttr container.Attributes, podAttr pod.Attributes) fileContent {
-	return fileContent{
-		PodName:                 podAttr.PodName,
-		PodUID:                  podAttr.PodUID,
-		PodNamespace:            podAttr.NamespaceName,
-		ClusterID:               podAttr.ClusterUID,
-		ContainerName:           containerAttr.ContainerName,
-		DeprecatedContainerName: containerAttr.ContainerName,
-		ImageName:               containerAttr.ToURI(),
+func fromAttributes(containerAttr container.Attributes, podAttr pod.Attributes, tenant string, isFullStack bool) fileContent {
+	fileContent := fileContent{
```

Comment:
i think i future, when we enable more linters, it will fail. 
```suggestion
	fc := fileContent{
```